quarkus-integration/quarkus/deployment/src/test/java/ai/timefold/solver/quarkus/TimefoldProcessorGeneratedGizmoSupplierTest.java

```diff
@@ -99,13 +99,17 @@ public interface DummyInterfaceEntity {
         @ShadowVariable(variableListenerClass = DummyVariableListener.class,
                 sourceEntityClass = TestdataEntity.class, sourceVariableName = "value")
         Integer getLength();
+
+        void setLength(Integer length);
     }
 
     @PlanningEntity
     public abstract static class DummyAbstractEntity {
         @ShadowVariable(variableListenerClass = DummyVariableListener.class,
                 sourceEntityClass = TestdataEntity.class, sourceVariableName = "value")
         abstract Integer getLength();
+
+        abstract void setLength(Integer length);
```

Comment:
@Christopher-Chianelli I tried to fix this myself, but I wasn't able to find the correct place to do this. Can I please ask you to:

- Make this fail fast if getter is declared, but setter is not. (And vice versa.)
- When the setter and getter is declared, to generate both of them. (Currently a test is complaining that a setter is missing.)

Technically, this is a bwd incompatible behavior. But in practice, I consider this is a bug - if a getter is declared, a setter must be declared to. It's either both or neither.